contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -256,22 +282,22 @@ library LiquidationLogic {
         DataTypes.ReserveData storage collateralReserve = reservesData[
             params.collateralAsset
         ];
-
         DataTypes.ReserveData storage liquidationAssetReserve = reservesData[
             params.liquidationAsset
         ];
         DataTypes.UserConfigurationMap storage userConfig = usersConfig[
             params.user
         ];
-        uint16 liquidationAssetReserveId = liquidationAssetReserve.id;
+
+        vars.liquidationAssetReserveId = liquidationAssetReserve.id;
         vars.liquidationAssetReserveCache = liquidationAssetReserve.cache();
+        liquidationAssetReserve.updateState(vars.liquidationAssetReserveCache);
```

Comment:
is `updateState` required here? seems already covered in supply logic

Reply:
seems not need here.